### DISCUSSION TITLE
[TAKE OUR SURVEY]: Share what you think about the community!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#69811] [Announcements]
Hi there! 😸 

We want to check in about your experience with github.com/community and invite you to provide thoughts and feedback on **[this survey](https://survey3.medallia.com/?kZXpVW-community)**! How's it going?

We’ve been using [GitHub Discussions](https://github.com/features/discussions) to power github.com/community as the official home for GitHub Q&A, show & tell, and feedback on GitHub products and are keen to hear your thoughts on the feature. 

What's working and where can we improve? Are you finding value here or getting your questions answered? What could take this experience up a notch? This 5-minute survey will help us better understand your experience. Your feedback will help inform our next iteration of this space, so please let us know what you think! We're excited to hear from you ✨

## **[Link to survey](https://survey3.medallia.com/?kZXpVW-community)**

**This survey will now close on October 29th, 2023** so be sure to participate to make your feedback heard.

Thank you!

### P.S. What's changed since the last survey?
The last time we ran this survey was about 6 months ago. Based on the feedback from that survey, we've made a few changes to the Community and Discussions, including:
* Continued iteration of back-end spam management
* Updated moderation policies and our Code of Conduct 
* [Filter answered discussions through the GraphQL API](https://github.blog/changelog/2023-10-10-filter-answered-discussions-through-the-graphql-api/)
* [Organize Discussions Categories into Sections](https://github.blog/changelog/2023-04-17-organize-discussions-categories-into-sections/)

**Comment by Pauljbern** [comment]
I've had identity theft for 4+ years as a double amputee below the knee it's landing me in the hospital four times last time broken hip broken femur before that more taken off my right leg. I don't know your world computers. I was a master electrician for 30 years, after I lost my house, my legs in my wife, she didn't like the new look that was nothing compared to identity theft in through the phones. The phone number talking on now was sent to me by AT&T arrived two weeks late. I've had nothing but problems. I've asked everyone, I found bug type 211 spyware in my analytics plus more bugs I just need help whatever you can do for me I would greatly appreciate this has gotten into my bank account. I've notified federal trade commission.

**Reply by Pauljbern** [reply]
Professionalism honesty

**Comment by Dew0Tgx** [comment]
thx

**Reply by 19920728** [reply]
hello can you help me with my phantom 4 drone. i need to remove the red forbidden zones

**Comment by Thrillseek** [comment]
Hello.sorry my usa talk is not the best.i have a issue whith mr messari,he need a proof that i gave your company millions of dollars innthe  last few months.can you plz comfirmit to him.im inna trubleiff not.knut arne johansen.

**Comment by samkit-jain** [comment]
What is the right format for providing the email and handle? I have tried multiple combinations but it always fails with the same error with no indication whatsoever of what constitutes a valid entry. 
![image](https://github.com/community/community/assets/15127115/bfda3a1e-3c78-4713-bf51-c2be0c1ff1a8)

**Reply by samkit-jain** [reply]
It submits when you just provide the email. In that case, the GitHub handle requirement is invalid.

**Reply by seyLu** [reply]
yup, that was confusing :/

**Reply by 19920728** [reply]
hello can you help me with my phantom 4 drone. i need to remove the red forbidden zones

**Reply by queenofcorgis** [reply]
Thank you so much for the feedback - we'll make sure this is updated going forward and we apologize for the confusing experience.

**Comment by Digitaltransform** [comment]
Saya akan mengikuti survey tapi saya bergembira dulu dong! Patut dirayakan ...cheers 🥂 karena Github komoniti telah memberikan badge pada profile saya, walau lama sekali profile github saya BALD.
Yang saya ingin tahu kan oh no, saya ingin tanyakan adalah perbedaan yang mendasar atas komunitas dan Perusahaan.
Bagaimana twntang wadah resmi sesuai aturan yang berlaku tentang komunitas tersebut apakah wadahnya Yayasan (NGO) atau suatu Lembaga yang bebas berdiri dimanapun secara global .
Jika bisnivjelas lah kita semua faham apa itu wadah bisnis tentu mengacu kepada firmayang sesuai dengan ....

**Comment by 19920728** [comment]
We want to check in about your experience with github.com/community and invite you to provide thoughts and feedback on [this survey](https://survey3.medallia.com/?kZXpVW-community)! How's it going?

We’ve been using [GitHub Discussions](https://github.com/features/discussions) to power github.com/community as the official home for GitHub Q&A, show & tell, and feedback on GitHub products and are keen to hear your thoughts on the feature.

What's working and where can we improve? Are you finding value here or getting your questions answered? What could take this experience up a notch? This 5-minute survey will help us better understand your experience. Your feedback will help inform our next iteration of this space, so please let us know what you think! We're excited to hear from you ✨

**Comment by 19920728** [comment]
hello can you help me with my phantom 4 drone. i need to remove the red forbidden zones

**Comment by andrew28497711-1** [comment]
> Hi there! 😸
> 
> We want to check in about your experience with github.com/community and invite you to provide thoughts and feedback on **[this survey](https://survey3.medallia.com/?kZXpVW-community)**! How's it going?
> 
> We’ve been using [GitHub Discussions](https://github.com/features/discussions) to power github.com/community as the official home for GitHub Q&A, show & tell, and feedback on GitHub products and are keen to hear your thoughts on the feature.
> 
> What's working and where can we improve? Are you finding value here or getting your questions answered? What could take this experience up a notch? This 5-minute survey will help us better understand your experience. Your feedback will help inform our next iteration of this space, so please let us know what you think! We're excited to hear from you ✨
> 
> ## **[Link to survey](https://survey3.medallia.com/?kZXpVW-community)**
> **This survey will now close on October 29th, 2023** so be sure to participate to make your feedback heard.
> 
> Thank you!
> 
> ### P.S. What's changed since the last survey?
> The last time we ran this survey was about 6 months ago. Based on the feedback from that survey, we've made a few changes to the Community and Discussions, including:
> 
> * Continued iteration of back-end spam management
> * Updated moderation policies and our Code of Conduct
> * [Filter answered discussions through the GraphQL API](https://github.blog/changelog/2023-10-10-filter-answered-discussions-through-the-graphql-api/)
> * [Organize Discussions Categories into Sections](https://github.blog/changelog/2023-04-17-organize-discussions-categories-into-sections/)

**Reply by Yogendr00** [reply]
Hi sir am new

**Comment by queenofcorgis** [comment]
Thanks everyone who participated!
